libs/watt/package/dropzone/watt-dropzone.ts

```diff
@@ -0,0 +1,175 @@
+    }
+
+    .accepted {
+      background: var(--watt-color-state-success-light);
+    }
+
+    .rejected {
+      background: var(--watt-color-state-danger-light);
+    }
+
+    .medium-emphasis {
+      color: var(--watt-on-light-medium-emphasis);
+    }
+
+    input[type='file'] {
+      display: none;
+    }
+  `,
+  template: `
+    <watt-field [label]="label()">
+      <ng-content />
+      <ng-content select="watt-field-error" ngProjectAs="watt-field-error" />
+      <ng-content select="watt-field-hint" ngProjectAs="watt-field-hint" />
+      <span class="wrapper">
+        <vater-stack
+          inset="0"
+          class="dropzone"
+          [class]="acceptState()"
+          (dragover)="handleDragOver($event)"
+          (drop)="handleDrop($event)"
+          (drop)="acceptState.set('indeterminate')"
+          (dragleave)="acceptState.set('indeterminate')"
+        >
+          <vater-stack center gap="xs">
+            <input
+              #input
+              type="file"
+              [multiple]="multiple()"
+              [accept]="accept().join(',')"
+              (change)="handleFiles(input.files)"
+            />
+            <span>{{ multiple() ? intl.promptMultiple : intl.prompt }}</span>
+            <span class="medium-emphasis">{{ intl.separator }}</span>
+            <watt-button size="small" variant="secondary" (click)="input.click()">
+              {{ multiple() ? intl.buttonMultiple : intl.button }}
+            </watt-button>
+          </vater-stack>
+        </vater-stack>
+      </span>
+    </watt-field>
+  `,
+})
+// eslint-disable-next-line @angular-eslint/component-class-suffix
+export class WattDropZone implements ControlValueAccessor {
+  intl = inject(WattDropZoneIntlService);
+
+  /** Whether the dropzone should accept multiple files. */
+  multiple = input(false, { transform: booleanAttribute });
+
+  /** Label for the dropzone. */
+  label = input<string>();
+
+  /** Comma-separated list of MIME types that the dropzone accepts. */
+  accept = input([], { transform: (value: MimeType) => value.split(',') as MimeType[] });
+  acceptState = signal<'indeterminate' | 'accepted' | 'rejected'>('indeterminate');
+
+  /** Emits when one or more files are selected. */
+  selected = output<File[]>();
+
+  handleFiles(files: FileList | null) {
+    if (files) {
+      this.selected.emit(Array.from(files));
+    }
+  }
+
+  handleDrop(event: DragEvent) {
+    if (!event.dataTransfer) return;
+    if (this.acceptState() === 'indeterminate') return;
+
+    // Prevent opening the file in the browser
+    event.preventDefault();
+
+    if (this.acceptState() === 'accepted') {
+      this.handleFiles(event.dataTransfer.files);
+    }
+  }
+
+  handleDragOver(event: DragEvent) {
+    if (!event.dataTransfer) return;
+
+    const items = Array.from(event.dataTransfer.items);
+    const accept = this.accept();
+
+    // Ignore non-file items such as strings
+    if (items.some((i) => i.kind !== 'file')) return;
```

Comment:
Question: It seems wrong, that just because one item is not a file, all are rejected? But its properly right :-)